insights/specs/datasources/compliance/compliance_ds.py

```diff
@@ -271,3 +278,46 @@ def compliance_unassign(broker):
         logger.error(err_msg)
         logger.debug(format_exc())
         sys.exit(constants.sig_kill_bad)
+
+
+@datasource(os_version, package_check, HostContext, timeout=0)
+def compliance_advisor_rule_enabled(broker):
+    try:
+        result = {}
+        insights_config = broker.get('client_config')
+        compliance = ComplianceClient(
+            os_version=broker[os_version], ssg_version=broker[package_check], config=insights_config
+        )
+        policies = compliance.get_system_policies()
+        if not policies:
+            logger.error(
+                "System is not associated with any policies. Assign policies using the Compliance web UI.\n"
+            )
+            raise SkipComponent
+        result['enabled_policies'] = policies
+        result['tailoring_policies'] = []
+        for policy in policies:
+            tailoring_content = compliance.fetch_tailoring_content(policy)
+            if tailoring_content:
+                tailoring_policy = {}
+                tailoring_policy['ref_id'] = policy['ref_id']
+                tailoring_policy['check_items'] = []
+                xml_root = ET.fromstring(tailoring_content)
+                pre_tag = xml_root.tag.split("Tailoring")[0]
+                profile_select_tag = pre_tag + 'Profile/' + pre_tag + 'select'
+                profile_select_info = xml_root.findall(profile_select_tag)
+                for item in profile_select_info:
+                    tailoring_policy['check_items'].append(item.attrib)
+                result['tailoring_policies'].append(tailoring_policy)
+        if not result['tailoring_policies']:
+            del result['tailoring_policies']
```

Comment:
How about:
```python
tailoring_policies = []


for ...
    ...
    tailoring_policies.append(tailoring_policy)
if tailoring_policies:
    result['tailoring_policies'] = tailoring_policies
```
instead of adding it then remove it when empty?